airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -738,6 +739,15 @@ private boolean getFailStatus(final StandardSyncOutput standardSyncOutput) {
       return true;
     }
 
+    // catch normalization failure reasons
+    final NormalizationSummary normalizationSummary = standardSyncOutput.getNormalizationSummary();
+    if (normalizationSummary != null && normalizationSummary.getFailures() != null &&
+        !normalizationSummary.getFailures().isEmpty()) {
+      workflowInternalState.getFailures().addAll(normalizationSummary.getFailures());
+      workflowInternalState.setPartialSuccess(true); // fail at normalization means replication successful (?)
```

Comment:
_it depends_

normalization might successfully update some of the final tables (in which case yes, partial success). But it could also fail to update any tables, in which case I'd call that a total failure. _Technically_ the sync maybe pushed data into the raw tables, but that's not useful for most users.

Reply:
I've removed the setPartialSuccess as true for ^ this reason + previously normalization failures **weren't** setting it to true.